appveyor.yml

```diff
@@ -21,7 +21,7 @@ init:
 
 install:
   - set PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
-  - python -m pip install -U pip
+  - python -m pip install -U pip setuptools
```

Comment:
I think we need to explicitly install mock for Python 2.7 on appveyor?

Reply:
True. Let me fix this.

Reply:
@joshuagl, do you happen to know how to install the dependency only for Python 2.7 builds on appveyor?

Reply:
Not off the top of my head, but I think we should be able to use `for.matrix.only/.except`:
https://www.appveyor.com/docs/build-configuration/#specializing-matrix-job-configuration

Reply:
Thanks. Maybe 932c025 works too.